distribution/ddf-common/src/main/resources/bin/get_property

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+PROPERTY_NAME=$1
+
+# Useful directories
+SCRIPTDIR=$(dirname $0)
+HOME_DIR=$(cd "${SCRIPTDIR}/.."; pwd -P)
+
+# Useful files
+PROPERTIES_FILE=$HOME_DIR/etc/system.properties
+
+# Check file exists where it is supposed to
+if [ ! -f $PROPERTIES_FILE ]; then
+    echo Cannot find file $PROPERTIES_FILE >&2
+    exit 1
+fi
+
+# Extract the value of a property from a Java properties file
+# This function does not handle multi-line properties
+# If the property does not exist, print  error message to stderr
+value=""
+value=`grep ^$PROPERTY_NAME= "$PROPERTIES_FILE" | cut -d = -f2`
+
+if [ $value ]; then
```

Comment:
if you are checking that the property is not unset and not equal to an empty string then do `if  [ ! -z "$value" ]; then`